sdk/storage/storage-blob/samples/javascript/iterators-blobs.js

```diff
@@ -110,7 +110,7 @@ async function main() {
   }
   // Gets next marker
   console.log("\tContinuation");
-  let marker = response.value.nextMarker;
```

Comment:
Is nextMarker still accessible from user side? Or it's actually a renamed property? If it's still accessible from user side, should we find some method to block that?

Reply:
It's renamed in the protocol layer so no longer accessible.